candi/bashible/bundles/centos/bootstrap.sh.tpl

```diff
@@ -40,6 +40,13 @@ if [ "${VERSION_ID}" == "8" ] ; then
   alternatives --set python /usr/bin/python3
 fi
 
+for FS_NAME in `egrep -v -e '(^[[:space:]]*$|^#)' /etc/fstab | awk '{ print $1 }'`; do
+  if  [[ `xfs_info $FS_NAME | grep --count ftype=0` -eq 1 ]]; then
```

Comment:
What if there isn't a xfs_info? and why don't just grep:
```suggestion
  if  xfs_info $FS_NAME | grep -q ftype=0; then
```

Reply:
I checked on centos 7/8/9 and rocky 9, looks like xfs_info exists in centos-like distros by default.

Fixed to grep.